app/src/main/java/org/mozilla/focus/fragment/onboarding/OnboardingFragment.kt

```diff
@@ -149,34 +141,24 @@ class OnboardingFragment : Fragment() {
 
     @Composable
     fun KeyFeatureCard(iconId: Int, titleId: Int, descriptionId: Int) {
-        ConstraintLayout(modifier = Modifier.fillMaxWidth()) {
-            val (image, title, description) = createRefs()
-            Image(
-                painter = painterResource(iconId),
-                colorFilter = ColorFilter.tint(color = focusColors.onboardingKeyFeatureImageTint),
-                contentDescription = getString(R.string.app_name),
-                modifier = Modifier.constrainAs(image) { top.linkTo(parent.top) }
```

Comment:
To fix the RTL bug we can only constraint the image to start where the parent does start.linkTo(parent.start)

![Screenshot 2022-03-17 at 10 49 39](https://user-images.githubusercontent.com/11731652/158772568-9bc1026f-7e89-4462-b165-958cfcdc995d.png)

Reply:
In this way we don't need to replace the constraint layout with a row and add paddings

Reply:
yes

Reply:
Can you please squash the commits?

Reply:
I squashed the commits